code/tests.py

```diff
@@ -0,0 +1,21 @@
+"""These tests can be run with pytest.
```

Comment:
It's wonderful to see tests being added. That said, I wonder if now is the time to test the dark visitors loading and merging code too?

Reply:
It would be nice to have that, but this is outside my current purvey.